dev/export-datameer-env.sh

```diff
@@ -0,0 +1,7 @@
+export __vHadoop='hadoop-2.7'
```

Comment:
why 2.7? Isn't spark-2.4.2 based on hadoop-2.6.5?

Reply:
you can choose between them, we chosed hadoop2.7 for the image we worked with

Reply:
It is.

Reply:
https://archive.apache.org/dist/spark/spark-2.4.2/
you can choose both